package.json

```diff
@@ -28,7 +28,7 @@
     "install-debug": "node-gyp rebuild --debug",
     "rebuild": "npm run lzz && node-gyp rebuild",
     "rebuild-debug": "npm run lzz && node-gyp rebuild --debug",
-    "test": "$(npm bin)/mocha --exit --slow=75 --timeout=2000",
+    "test": "mocha --exit --slow=75 --timeout=5000",
```

Comment:
I don't like the idea of people needing a global `mocha` installation to run `better-sqlite3` tests. Is there an alternative to this that will work for Windows? If not, I'd prefer to have the Windows-specific command secluded to `appveyor.yml`

Reply:
**This doesn't require a global mocha**. Yarn (and newer npm) will look for the binary in `node_modules/.bin`.

Reply:
Wow, that's news to me!

Reply:
Yeah, it's pretty nice. It means you can't just copy-paste the npm script to the terminal and expect it to work, but it does makes your script more terse.